osbs/cli/main.py

```diff
@@ -625,7 +625,10 @@ def cli():
 
     get_login_parser = subparsers.add_parser(str_on_2_unicode_on_3('login'),
                                              help='perform login and store token for later use')
-    get_login_parser.add_argument('TOKEN', help='token to be used for login')
+    get_login_parser.add_argument('--token', help='token to be used for login '
+                                                  '(takes precedence over username and password)', action="store")
```

Comment:
Now that we report an error, the help for `--token` needn't mention username and password.